### PR TITLE
feat: replace mumbai for amoy

### PR DESCRIPTION
**Pull request by juanmahidalgo**

### CODE REVIEW

File: src/ports/snapshot/constants.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const strategiesByChainId = {
         name: 'multichain',
         graphs: {
           [ChainId.ETHEREUM_GOERLI]: 'https://api.thegraph.com/subgraphs/name/decentraland/blocks-ethereum-goerli',
-          [ChainId.MATIC_MUMBAI]: 'https://api.thegraph.com/subgraphs/name/decentraland/blocks-matic-mumbai'
+          [ChainId.MATIC_AMOY]: 'https://api.thegraph.com/subgraphs/name/decentraland/blocks-matic-amoy'
```
Comment (cyaiox) [inline src/ports/snapshot/constants.ts]:
```suggestion
          [ChainId.MATIC_AMOY]: 'https://api.studio.thegraph.com/query/49472/blocks-matic-amoy/version/latest'
```

File: src/ports/snapshot/constants.ts [hunk 2]
```diff
@@ -135,7 +135,7 @@ export const strategiesByChainId = {
         name: 'multichain',
         graphs: {
           [ChainId.ETHEREUM_SEPOLIA]: 'https://api.studio.thegraph.com/query/49472/blocks-ethereum-sepolia/version/latest',
-          [ChainId.MATIC_MUMBAI]: 'https://api.thegraph.com/subgraphs/name/decentraland/blocks-matic-mumbai'
+          [ChainId.MATIC_AMOY]: 'https://api.thegraph.com/subgraphs/name/decentraland/blocks-matic-amoy'
```
Comment (cyaiox) [inline src/ports/snapshot/constants.ts]:
```suggestion
          [ChainId.MATIC_AMOY]: 'https://api.studio.thegraph.com/query/49472/blocks-matic-amoy/version/latest'
```

File: .env.spec [hunk 3]
```diff
@@ -3,7 +3,7 @@ HTTP_SERVER_HOST=localhost
 PG_COMPONENT_PSQL_CONNECTION_STRING=postgres://testuser:testpassword@localhost:5432/marketplace
 PG_COMPONENT_PSQL_SCHEMA=favorites
 
-COLLECTIONS_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/collections-matic-mumbai
+COLLECTIONS_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/collections-matic-amoy
```
Comment (cyaiox) [inline .env.spec]:
```suggestion
COLLECTIONS_SUBGRAPH_URL=https://subgraph.decentraland.org/collections-matic-amoy
```

Comment (juanmahidalgo) [reply .env.spec]: done!

File: src/ports/snapshot/constants.ts (hunk 2 above)
Comment (juanmahidalgo) [reply src/ports/snapshot/constants.ts]: done!

File: src/ports/snapshot/constants.ts (hunk 1 above)
Comment (juanmahidalgo) [reply src/ports/snapshot/constants.ts]: done!

**Review by cyaiox — APPROVED** [review]

**Pull request merged**